src/structures/Message.js

```diff
@@ -298,47 +328,6 @@ class Message extends Base {
     return typeof this.content !== 'string' || !this.author;
   }
 
-  /**
-   * Updates the message and returns the old message.
-   * @param {APIMessage} data Raw Discord message update data
-   * @returns {Message}
-   * @private
-   */
-  patch(data) {
-    const clone = this._clone();
-
-    if (data.edited_timestamp) this.editedTimestamp = new Date(data.edited_timestamp).getTime();
```

Comment:
This isn't set anywhere anymore, unless I missed it

Reply:
It is, just not in the diff since I didn't need to change it
https://github.com/discordjs/discord.js/pull/6071/files#R178